codesearch/index/mmap_windows.go

```diff
@@ -33,5 +33,25 @@ func mmapFile(f *os.File) mmapData {
 		log.Fatalf("MapViewOfFile %s: %v", f.Name(), err)
 	}
 	data := (*[1 << 30]byte)(unsafe.Pointer(addr))
-	return mmapData{f, data[:size]}
+	return mmapData{f, data[:size], data[:]}
 }
+
+func unmmapFile(m *mmapData) error {
+	err := syscall.UnmapViewOfFile(uintptr(unsafe.Pointer(&m.d[0])))
+	if err != nil {
+		log.Fatalf(err)
```

Comment:
@kellegous Ah true, we should at least keep consistency with the other unmmap implemenetations. I'll change it